src/control/server/init/setup_spdk.sh

```diff
@@ -17,9 +17,12 @@ else
 	PCI_WHITELIST="$_PCI_WHITELIST" NRHUGE="$_NRHUGE" \
 	TARGET_USER="$_TARGET_USER" "$scriptpath"
 
-	chmod 777 /dev/hugepages
-	chmod 666 /dev/uio*
-	chmod 666 /sys/class/uio/uio*/device/config
-	chmod 666 /sys/class/uio/uio*/device/resource*
+	eet -e
+
+	chown -R "$_TARGET_USER" /dev/hugepages /dev/uio* \
+		/sys/class/uio/uio*/device/config \
+		/sys/class/uio/uio*/device/resource*
+
+	set e
```

Comment:
Typo?

```suggestion
	set +e
```

But more importantly there doesn't seem to be any reason to alter the `set -e` state at this point.

FWIW, it's good form to run entire scripts with `set -e` enabled and handle the exceptions/errors where they might happen using `if ...; then` and/or `||`, etc. to prevent errors from being uncaught.